install_config/aggregate_logging.adoc

```diff
@@ -96,14 +85,14 @@ $ oc project logging
 [NOTE]
 ====
 Specifying a non-empty
-link:../admin_guide/managing_projects.html#using-node-selectors[node
+xref:../admin_guide/managing_projects.html#using-node-selectors[node
 selector] on the project is not recommended, as this would restrict
 where Fluentd can be deployed. Instead, specify node selectors for the
-deployer to be applied on your other deployment configurations.
+deployer to be applied to your other deployment configurations.
 ====
 
-. Create a link:../dev_guide/secrets.html[secret] for the deployer. The secret
-can be configured specifically for your instance of the deployer.
+. Create a xref:../dev_guide/secrets.html#[secret] for the deployer. The secret
+can be configured specifically for the deployer.
```

Comment:
The contents of the secret are all optional (will be generated if not
supplied). The secret itself is not... yet.

On Mon, Jun 6, 2016 at 12:19 AM, Brice Fallon-Freeman <
notifications@github.com> wrote:

> In install_config/aggregate_logging.adoc
> <https://github.com/openshift/openshift-docs/pull/2190#discussion_r65836064>
> :
>
> >  ====
> >
> > -. Create a link:../dev_guide/secrets.html[secret] for the deployer. The secret
> > -can be configured specifically for your instance of the deployer.
> > +. Create a xref:../dev_guide/secrets.html#[secret] for the deployer. The secret
> > +can be configured specifically for the deployer.
>
> I'll admit, i was questioning the second sentence myself, but you reworded
> into what it is today, so I thought it was relevant. I largely kept with
> what you suggested, but took out "optional". Does that mean the contents of
> the secret are optional? I feel that requires a lot of explanation. Unless
> you mean the secret itself is optional? In this case, we need to change up
> a lot of stuff I think...
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/openshift-docs/pull/2190/files/4cb973a5783c46612b33a6b2d66fe4975badbd32#r65836064>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABz-tD3uNMPcBSGiOORM97VfiKNqEA-ks5qI5_XgaJpZM4IrNCq>
> .
>